scrapy/pipelines/media.py

```diff
@@ -27,6 +27,21 @@ def __init__(self, spider):
     def __init__(self, download_func=None):
         self.download_func = download_func
 
+
+    def _key_for_pipe(self, key, base_class_name=None):
+        """
+        >>> result = MediaPipeline()._key_for_pipe("IMAGES")
+        >>> assert result == "IMAGES"
```

Comment:
good point, I need to write more doctests. fixed in 9818c97f6e9ac721bb136e4d2a4bfe1ffc8cc047